src/navigations/HomeBottomTabs.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-use-before-define */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `/* eslint-disable no-use-before-define */` at the top of your file is not recommended. It's better to fix these issues than to ignore them, as they can lead to bugs in your code. If you're using functions or variables before they are defined, consider rearranging your code so that definitions happen before usage.



<!-- This is an auto-generated comment by OSS CodeRabbit -->